apis/exoscale/v1/postgresql_types.go

```diff
@@ -15,11 +15,8 @@ type PostgreSQLParameters struct {
 	Maintenance MaintenanceSpec `json:"maintenance,omitempty"`
 	Backup      BackupSpec      `json:"backup,omitempty"`
 
-	// +kubebuilder:validation:Enum=ch-gva-2;ch-dk-2;de-fra-1;de-muc-1;at-vie-1;bg-sof-1
-	// +kubebuilder:validation:Required
```

Comment:
I would keep the Required marker here. In case the Zone type is used in another Go project, it doesn't also become required (e.g. in the XRD code generator)